service.mailnotifiers/notifier.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+"""
+kodi service for display number of mail on home screen.
+"""
+
+import xbmc, xbmcgui
+import xbmcaddon
+import poplib, imaplib
+import time
+
+__author__     = "Senufo"
```

Comment:
in future updates, please remove the \__varname\__ style labels. these are reserved as internals in python.